HISTORY.rst

```diff
@@ -14,3 +14,8 @@ History
 * First public release (beta).
 
 
+0.3.0-dev (2023-08-08)
+----------------------
+
+* Added automatic COG filter to narrow down available collection (Issue #66)
+* Added interact by point tool to get coordinates and raster values at a location (Issue #64)
```

Comment:
Should we cut the release before we merge this though?

Reply:
Looks like we dont need to cut a tag just yet but this starts the change log atleast for when we do cut for this tag 🙌🏼